filters/f_output_chain.c

```diff
@@ -114,7 +116,8 @@ static void check_in_format_change(struct mp_user_filter *u,
             // But a common case is enabling HW decoding, which
             // might init some support of them in the VO, and update
             // the VO's format list.
-            update_output_caps(p);
+            bool query_formats = !strcmp(u->name, "convert");
+            update_output_caps(p, query_formats);
```

Comment:
Instead of adding a bool to the function, how about only calling it if it's the convert filter? It seems like all other usages of `update_output_caps` always use `true`.

Reply:
Good call. I lost sight of the bigger picture there.